#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_default_config.dpatch" by David Gil <dgil@telefonica.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian modifications to fix the pgsql name

@DPATCH@
--- acidbase-1.2.7.orig/base_conf.php.dist	2006-04-03 12:04:04.000000000 +0200
+++ acidbase-1.2.7/base_conf.php.dist	2006-04-03 12:09:57.000000000 +0200
@@ -92,6 +92,9 @@
 ###### Begin of variables configured through dbconfig-common
 
 require('/etc/acidbase/database.php');
+# Adjust dbconfig-common names
+if ($DBtype =='pgsql') 
+    $DBtype = 'postgres';
 
 ###### End of variables configured through dbconfig-common
 
